src/dashboard/home-dashboard.component.tsx

```diff
@@ -8,7 +8,7 @@ export default function HomeDashboard(props: HomeDashboardProps) {
       <section className={styles.mainSection}>
         <Link
           to={props.match.url + "/patient-search"}
-          className={`omrs-unstyled ${styles.patientSearchLink}`}
+          className="omrs-link omrs-outlined-action omrs-rounded"
         >
```

Comment:
@joeldenning according to figma, this isn't a button or link but a text input

Reply:
From discussions with @gregoryjschmidt, it appears as an input that's focused on page load. When a user starts typing it then takes you to the new page, did this change?

Reply:
👍